root/ntuple/ntuple-makeproject/write_rntuple.cxx

```diff
@@ -0,0 +1,15 @@
+#include "ntuple_makeproject_header.h"
+#include <ROOT/RNTupleModel.hxx>
+#include <ROOT/RNTupleWriter.hxx>
+
+using RNTupleModel = ROOT::Experimental::RNTupleModel;
+using RNTupleWriter = ROOT::Experimental::RNTupleWriter;
+
+int main()
+{
+   auto model = RNTupleModel::Create();
+   auto fldStlEvent = model->MakeField<MySTLEvent>("test");
+
+   auto ntuple = RNTupleWriter::Recreate(std::move(model), "events", "ntuple_makeproject_stl_example_rntuple.root");
+   ntuple->Fill();
+}
```

Comment:
```suggestion
    return 0;
}
```

Reply:
Why is this necessary?

Reply:
Otherwise the return value and thus the exit code is random.

Reply:
And some compiler will issue warnings about the missing return.